Homeworks/Homework_4/Mika_Dominik.R

```diff
@@ -0,0 +1,17 @@
+library(ggplot2)
+library(data.table)
+
+
+
+ggdt <- function(data, x, y, type = geom_point, theme = theme()){
```

Comment:
chodziło o 3 kropki, ale to też pozwala na wykonanie polecenia, tyle że brakuje przykładowe wywołania, które to pokazuje